### PR TITLE
Add voms config files (added from opensciencegrid/osg-vo-config)

### PR DESCRIPTION
**Pull request by oshadura**
Add voms config files (synced from lxplus)
Should help with the [next issue](https://atlas-talk.sdcc.bnl.gov/t/unable-to-initialte-voms-proxy-or-access-cvmfs/179/5)

### CODE REVIEW

**Comment by jthiltges** [comment]
How about instead pulling the voms configs in the Dockerfile?

You could pull a specific release:
```
curl -L https://github.com/opensciencegrid/osg-vo-config/archive/refs/tags/release-124.tar.gz | tar -xz --strip-components=1 --directory=/etc/grid-security
```

Or just follow master, to avoid needing future changes:
```
curl -L https://github.com/opensciencegrid/osg-vo-config/archive/refs/heads/master.tar.gz | tar -xz --strip-components=1 --directory=/etc/grid-security
```

**Comment by oshadura** [comment]
@jthiltges good idea! but what about /etc/vomses files?

**Comment by jthiltges** [comment]
The command includes that file too. But now that I double-check, it includes too much. (Everything in the osg-vo-config repo!) This updated command should do the right thing:

```
curl -L https://github.com/opensciencegrid/osg-vo-config/archive/refs/heads/master.tar.gz | \
  tar -xz --strip-components=1 --directory=/tmp/x/etc/grid-security */vomses */vomsdir
```

**Comment by oshadura** [comment]
Thanks! Yeah I noticed that it has even GH actions :)

**Comment by jthiltges** [comment]
I do like the idea of pulling an upstream source for this info. There have been some recent changes to DN formats (and they probably won't be the last).

[SOFTWARE-4956](https://opensciencegrid.atlassian.net/browse/SOFTWARE-4956) is the most recent example.

File: docker/Dockerfile.cc-base-ubuntu [hunk 1]
```diff
@@ -237,6 +237,10 @@ USER root
 COPY condor/condor_config /etc/condor/
 COPY condor/config.d  /etc/condor/config.d/
 
+RUN curl -L https://github.com/opensciencegrid/osg-vo-config/archive/refs/heads/master.tar.gz | \
+    tar -xz --strip-components=1 --directory=/etc/grid-security */vomses */vomsdir && \
```
Comment (jthiltges) [inline docker/Dockerfile.cc-base-ubuntu:241]: When I tested on Fedora, no issues. But it seems Ubuntu 20.04 expects the `--wildcards` argument.

```suggestion
    tar -xz --strip-components=1 --directory=/etc/grid-security --wildcards */vomses */vomsdir && \
```

**Review by jthiltges — CHANGES_REQUESTED** [review]

**Review by jthiltges — APPROVED** [review]
LGTM!

**Pull request merged**